cpp/include/cuvs/neighbors/vamana.hpp

```diff
@@ -60,7 +60,7 @@ struct index_params : cuvs::neighbors::index_params {
    * literature **/
   uint32_t visited_size = 64;
   /** Number of Vamana vector insertion iterations (each iteration inserts all vectors). */
-  uint32_t vamana_iters = 1;
+  float vamana_iters = 1.0;
```

Comment:
How does a fraction of an iteration work here?

Reply:
Good question - this is the main workaround for any minor recall differences from CPU to GPU. It re-iterates over the dataset and stops after the fraction is reached. For example, with at 1.1, it inserts the entire dataset, then re-inserts the first 10% of the data to improve the graph quality. Since the early vectors were inserted into an empty graph, re-inserting them has the biggest impact on recall.

Reply:
Added a check in `vamana_build()` that it's >1.0.